CITATION.cff

```diff
@@ -0,0 +1,55 @@
+cff-version: 1.2.0
+title: DLA-Future
```

Comment:
I've now updated the author list on zenodo to match the one here. While doing that I noticed one more discussion point: the title.

The auto-generated one by zenodo is `eth-cscs/DLA-Future: DLA-Future 0.4.0`, i.e. `<org>/<repo>: <maybe-description-but-not-sure> <version>`.

I've currently used the generic `DLA-Future` here. I think we probably have at least the following options:
- `DLA-Future`
- `DLA-Future <version>`
- `DLA-Future <version>: <description>`
- `eth-cscs/DLA-Future: <description> <version>`
and combinations of them.

I think my preference would be simply `DLA-Future <version>`. Do you have a preference?

Note that zenodo does have a "Description" textbox as well, which is currently populated with the release notes.

Reply:
`DLA-Future <version>` is my preferred choice as well.

Reply:
Updated to `DLA-Future 0.4.0`